.travis.yml

```diff
@@ -19,6 +19,9 @@ install:
 
 script:
   - go get -t -v ./...
-  - diff -u <(echo -n) <(gofmt -d .)
-  - if [[ "$LATEST" = true ]]; then go vet .; fi
+  - |
+    if [[ "$LATEST" = true ]]; then
+      diff -u <(echo -n) <(gofmt -d .)
+      go vet .
+    fi
   - go test -v -race ./...
```

Comment:
Can you rebase this against master - it will remove this

Reply:
I rebased, but I'm not sure what you mean about it removing this?

Reply:
Alright rebased again should be deleted now